File: Zacchua-cs50-problems-2020-x-sentimental-mario-more/mario.py
from sys import exit
from cs50 import get_int

while True:
    h = get_int("height: ")
    if (h < 9 and h > 0):
        break

count = 1
for x in range(h):
    print(" " * (h-count), end = "")
    print("#" * count, end = "")
    print("  ", end = "")
    print("#" * count, end = "")
    print(" " * (h-count))
    count += 1

File: Zacchua-cs50-problems-2019-x-mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int get_positive_int (string prompt);

int main(void)
{
  int h = get_positive_int ("height:");
  for (int j=1; j<=h; j++)
    {
      for (int l=8; l>j; l--)
      {
          printf(" ");
      }
      for (int k=0; k<j; k++)
      {
          printf ("#");
      }
        printf ("\n");
    }
}

int get_positive_int (string prompt)
{
    int i;
    do
    {
        i = get_int ("%s", prompt);
    }
    while (i<1 || i>8);
    return i;        
}
